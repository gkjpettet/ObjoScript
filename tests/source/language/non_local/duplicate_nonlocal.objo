class Foo {
bar() {
var a = "value"
var a = "other" # Expect error as duplicate variable in same scope.
}
}