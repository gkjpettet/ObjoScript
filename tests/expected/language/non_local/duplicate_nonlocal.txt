There is already a variable with this name (a) in this scope.